.cirrus.yml

```diff
@@ -73,17 +73,24 @@ env:
 # Linux EOL timelines: https://linuxlifecycle.com/
 # Fedora (~13 months): https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle
 
+fedora33_task:
+  container:
+    # Fedora 33 EOL: Around November 2022
+    dockerfile: ci/fedora-33/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  << : *CI_TEMPLATE
+
 fedora32_task:
   container:
     # Fedora 32 EOL: Around May 2021
     dockerfile: ci/fedora-32/Dockerfile
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
 
-centos7_task:
+fedora31_task:
   container:
-    # CentOS 7 EOL: June 30, 2024
-    dockerfile: ci/centos-7/Dockerfile
+    # Fedora 31 EOL: Nov 24 2021
```

Comment:
Fedora 31 seems to actually be EOL in November 2020 (in 2 weeks) ?  May not be worth adding then.

Reply:
Yeah ... I was torn about it myself. I mainly added it to see if anything would break in CI! We can certainly skip it, too.

Reply:
I'll just leave it in for now. Easy enough to remove in a few weeks.